### ISSUE TITLE
BOTTOM() requires a tag key argument to return results

### ISSUE THREAD

**Issue by rkuchan** [influxdata/influxdb#8474]
### Bug report

Ubuntu, Nightly InfluxDB

__Steps to reproduce:__

#### 1. Write some data
```
> insert icecream,flavor=mint servings=1
> insert icecream,flavor=chocolate servings=2
> insert icecream,flavor=mint servings=3
> SELECT * FROM "icecream"
name: icecream
time                           flavor    servings
----                           ------    --------
2017-06-11T21:49:44.342962523Z mint      1
2017-06-11T21:49:53.831665866Z chocolate 2
2017-06-11T21:49:59.266227535Z mint      3
```
#### 2. Run a `BOTTOM()` and `TOP()` query with no tag argument
`BOTTOM()` returns nothing, but `TOP()` returns results.
```
> SELECT BOTTOM("servings",1) FROM "icecream"
>
```
```
> SELECT TOP("servings",1) FROM "icecream"
name: icecream
time                           top
----                           ---
2017-06-11T21:49:59.266227535Z 3
```
#### 3. Run a `BOTTOM()` and `TOP()` query with a tag outside the parentheses
`BOTTOM()` returns nothing, but `TOP()` returns results.
```
> SELECT BOTTOM("servings",1),"flavor" FROM "icecream"
>
```
```
> SELECT TOP("servings",1),"flavor" FROM "icecream"

name: icecream
time                           top flavor
----                           --- ------
2017-06-11T21:49:59.266227535Z 3   mint
```
#### 4. Run a `BOTTOM()` and `TOP()` query with a tag argument
Both `BOTTOM()` and `TOP()` return results.
```
> SELECT BOTTOM("servings",flavor,1) FROM "icecream"

name: icecream
time                           bottom flavor
----                           ------ ------
2017-06-11T21:49:44.342962523Z 1      mint
```
```
> SELECT TOP("servings",flavor,1) FROM "icecream"

name: icecream
time                           top flavor
----                           --- ------
2017-06-11T21:49:59.266227535Z 3   mint
```

__Expected behavior:__ 
I expected the `BOTTOM()` queries in steps two and three to return results.

__Actual behavior:__
The `BOTTOM()` queries in steps two and three return nothing.

**Comment by jsternberg** [comment]
This might be an old nightly. Can you check when the nightly was last built? There was a bug that got fixed in #8442.

**Comment by rkuchan** [comment]
You're right - sorry about that. Thanks for getting back to me!
